pkgs/development/lua-modules/overrides.nix

```diff
@@ -877,12 +877,6 @@ in
   });
 
   vusted = prev.vusted.overrideAttrs (_: {
-    postConfigure = ''
-      cat ''${rockspecFilename}
-      substituteInPlace ''${rockspecFilename} \
-        --replace-fail '"luasystem = 0.2.1",' "'luasystem >= 0.2',"
-    '';
-
```

Comment:
Newest rockspec is more lenient and just checks for a minimum version that matches what we have in nixpkgs.